test/test3.html

```diff
@@ -1,11 +1,3 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Without this header the file is not a valid web page.

Reply:
I don't think this package should be generating a web page.  Generating html code blocks that can embedded elsewhere is better.